src/IconsAndAssets/Icons/Github.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Github() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="16"
+      height="16"
+      fill="currentColor"
+      className="bi bi-github"
+      viewBox="0 0 16 16"
```

Comment:
You have done an impressive work but try not to use inline styling.

Reply:
It was just Svg so used them nothing much.